### PR TITLE
Keep upto date section

### PR DESCRIPTION
**Pull request by abygeorge7**
for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -329,6 +329,64 @@ <h3>Create better cloud outcomes with hands-on experience</h3>
             </div>
         </div>
     </section>
+    <!--keep uptodate section-->
+    <section class="keep-up-to-date-section">
+
```
Comment (BenjaminSao) [inline index.html:334]: Useless newline

File: index.html [hunk 2]
```diff
@@ -329,6 +329,64 @@ <h3>Create better cloud outcomes with hands-on experience</h3>
...
+        <div class="keep-up-to-date-heading">
+            <h6 class="title-pink">KEEP UP TO DATE</h6>
+            <h3 class="title-sub">Discover more about Pluralsight</h3>
+        </div>
+        <div class="social-items-all">
```
Comment (BenjaminSao) [inline index.html]: Use social-items. Plural denotes that there is multiple social item

File: index.html [hunk 3]
```diff
@@ -329,6 +329,64 @@ <h3>Create better cloud outcomes with hands-on experience</h3>
...
+            <div class="social-item">
+                <div>
```
Comment (BenjaminSao) [inline index.html:341]: Useless div

File: index.html [hunk 4]
```diff
@@ -329,6 +329,64 @@ <h3>Create better cloud outcomes with hands-on experience</h3>
...
+                    <img class="social-image-size"
+                        src="https://www.pluralsight.com/content/pluralsight/en/jcr:content/main/generic_block_135160/parsys/columns/column-parsys-1/flex_block/parsys/flex_open_block_copy_383891725/parsys/image/image-res.img.5eb03cae-1f8e-4ae5-bcd5-e583792df242.jpg"
+                        alt="">
+                </div>
+                <div class="">
```
Comment (BenjaminSao) [inline index.html]: Do you know what's wrong here?

**Review by BenjaminSao — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (abygeorge7) [reply index.html]: Yes ..deleted the class syntax .but those div are needed right?

Comment (abygeorge7) [reply index.html]: to make this block level element

File: index.html (hunk 3 above)
Comment (abygeorge7) [reply index.html:341]: those two divs , one for the image .and then for wrapping h and p tags

**Review by BenjaminSao — APPROVED** [review]

**Pull request merged**